src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -0,0 +1,178 @@
+import spacy
+import pandas as pd
+import argparse
+import re
+import csv
+
+def preProcessing(file, clutter_flag):
+    with open(file, 'r') as f:
+        content = f.read()
+    df = pd.read_json(content, orient='index')
+    df['content'] = df['content'].str.lower()
+    copyrightDeactivationMain(df, clutter_flag)
+
+def copyrightDeactivationMain(df, clutter_flag):
+    hit_index = []
+    clutter_flag = 0
+    nlp = spacy.load("en_core_web_sm")
+    ## Iterating through each row and doing preprocessing over it.
+    ## Picking out the manual tags from the csv and putting them into seperate column "Original Tag"
+    for index, row in df.iterrows():
+        text = df.loc[index, 'content']
+        doc = nlp(text)
+
+        if type(text) == float:
+            continue
+        
+        ## Lemmatization
+        lemma_list = []
+        for token in doc:
+            lemma_list.append(token.lemma_)
+        
+        # Filter the stopword
+        filtered_sentence =[] 
+        for word in lemma_list:
+            lexeme = nlp.vocab[word]
+            if lexeme.is_stop == False:
+                filtered_sentence.append(word)
+        
+        # Remove punctuation
+        punctuations="?:!.,;"
+        for word in filtered_sentence:
+            if word in punctuations:
+                filtered_sentence.remove(word)
+
+        ## List joining and Filtering (c) and copyright unicode symbol
+        list_of_copyrights = " ".join(map(str,filtered_sentence))
+        substring = "( c )"
+        cp_symbol = '\xa9' ##Unicode for copyright Symbol
+        
+        if "copyright" not in list_of_copyrights:
+            if substring in list_of_copyrights:
+                list_of_copyrights = list_of_copyrights.replace(substring, "copyright")
+            
+            elif cp_symbol in list_of_copyrights:
+                list_of_copyrights = list_of_copyrights.replace(cp_symbol, "copyright")
+                
+        if substring in list_of_copyrights:
+            list_of_copyrights = list_of_copyrights.replace(substring, "copyright")
+
+        elif cp_symbol in list_of_copyrights:
+            list_of_copyrights = list_of_copyrights.replace(cp_symbol, "copyright")
+    
```

Comment:
One additional step can be added here to replace `"copyright copyright"` with `"copyright"`?
For example, if the input is like `Copyright (C) 2021 Kaushlendra`, it will be normalized to `copyright copyright 2021 kaushlendra`.

But it will be required only if tagging is being effected because duplicate copyright word.

Reply:
No, It has no impact on the NER and POS calculation!

Reply:
Okay, then we can leave it.